packages/squiggle-lang/__tests__/ReducerProject/ReducerProject_test.res

```diff
@@ -0,0 +1,176 @@
+@@warning("-44")
```

Comment:
Good call with this. I assume this stops this warning in this file? I didn't realize that was a thing.

Reply:
Exactly.   Jest implementation overrides '==' which leads to unnecessary warnings.